bazel/external/rules_rust.patch

```diff
@@ -1,5 +1,4 @@
 # https://github.com/bazelbuild/rules_rust/pull/1315
-
```

Comment:
Nit: unnecessary line removal in a file that's otherwise not touched in this PR.